### ISSUE TITLE
[BUG] Anthropic Claude 3 Error Message -   json {   "error": true,   "message": "Failed to fetch" } 

### ISSUE THREAD

**Issue by niall-ofiz** [ChatGPTNextWeb/ChatGPT-Next-Web#4687]
### Bug Description

Any chat requests using any of the models: Sonnet, Haiku or Opus result in the error message:

```json
{
  "error": true,
  "message": "Failed to fetch"
}
```

### Steps to Reproduce

In settings:
Model Provider: Anthropic
Endpoint Address: https://api.anthropic.com
Anthropic API Key: Entered (verified working elsewhere)
API Version: 2023-06-01
Custom Models: left blank
Model: claude-opus-3-20240229(Anthropic)
All other settings as per default.

### Expected Behavior

Normal chat response from server.

### Screenshots

![Claude](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/155590729/7637ed24-993e-47b1-8a4c-10f7239021f7)

### Deployment Method

- [ ] Docker
- [ ] Vercel
- [ ] Server

### Desktop OS

Windows 10

### Desktop Browser

Firefox

### Desktop Browser Version

Windows Version 2.12.3

**Comment by fred-bf** [comment]
Are you using desktop app of NextChat?

**Comment by niall-ofiz** [comment]
Yes, Windows 10 and Ubuntu Desktop. Both Version 2.12.3. Same error on both installations.

**Comment by niall-ofiz** [comment]
> Are you using desktop app of NextChat?

Yes, Windows 10 and Ubuntu Desktop. Both Version 2.12.3. Same error on both installations.

**Comment by JesseMccree008** [comment]
i have the same problem

**Comment by phengite** [comment]
i have the same problem too.

**Comment by crivera8602** [comment]
Also having issue on windows 11 desktop

**Comment by f1h0** [comment]
{
  "error": true,
  "message": "empty response from server"
}

**Comment by f1h0** [comment]
** what happened Who can say **

**Comment by Lyinsun** [comment]
change your proxy

**Comment by llabusch93** [comment]
Having the same issue. Any resolve soon?

**Comment by dupinderdhiman** [comment]
I am having the same issue. Is there any resolve soon?

**Comment by m0rg5** [comment]
I stopped using because of this with Anthropic API, last update seemed to fix then mid chat (~50 msgs) got the same error again. Can't find a fix (new API's added, changed max tokens) and have had to stop using NextChat again. Again: Claude was working and then stopped mid chat with no change to settings.

```
{
  "error": true,
  "message": "empty response from server"
}
```

**Comment by linnnff** [comment]
我遇到了同样的问题。能尽快解决吗？

**Comment by hexsean** [comment]
I'm experiencing the same problem. Can you fix it as soon as possible? I'm also experiencing this issue on the web side, the same api and configuration works fine on the desktop program, but not on the web side!

**Comment by EricTLO** [comment]
+1 still problem
